Demo/Measures/ColorectalCancerScreeningsFHIR-0.0.003.g.cs

```diff
@@ -688,19 +689,20 @@ private IEnumerable<Condition> Total_Colectomy_Condition_Value()
 		var k_ = AdvancedIllnessandFrailtyExclusionECQMFHIR4_5_17_000.Advanced_Illness_and_Frailty_Exclusion_Not_Including_Over_Age_80();
 		var l_ = context.Operators.Or(j_, k_);
 		var m_ = this.Patient();
-		var n_ = context.Operators.Convert<CqlDate>(m_?.BirthDateElement?.Value);
+		var n_ = context.Operators.Convert<CqlDateTime>(m_?.BirthDateElement?.Value);
```

Comment:
Is this expected? A CqlDate becoming a CqlDateTime? Do we chose another overload?

Reply:
Fixed in latest PR